{"name": "test_login_failed[data1]", "status": "passed", "parameters": [{"name": "data", "value": "{'name': '登录功能-正常登录-用户名错误', 'username': '财友qfy46', 'password': 'xxx123456'}"}], "start": 1609584720249, "stop": 1609584721236, "uuid": "ec180ebb-3b02-4847-8b56-db702eeb8141", "historyId": "28fbee3f7849e5c692210a9f12e7fe96", "testCaseId": "130e6dce60bc4ef9d882c5f0465dc1d6", "fullName": "cases.test_login.TestLogin#test_login_failed", "labels": [{"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "PC-20200718ELAI"}, {"name": "thread", "value": "6176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}